NEWS.md

```diff
@@ -1,3 +1,22 @@
+## synapser 0.10
+___
+
+### Improvements
+
+* Login using an access token is now supported, e.g. `synLogin(authToken=token)`
+* The following additional functions are exposed:
+    + [synCreateExternalS3FileHandle](../reference/synCreateExternalS3FileHandle.md)
+    + [synCreateS3StorageLocation](../reference/synCreateS3StorageLocation.md)
+    + [synCreateSnapshotVersion](../reference/synCreateSnapshotVersion.md)
+    + [synGetStsStorageToken](../reference/synGetStsStorageToken.md)
+    + [synIsCertified](../reference/synIsCertified.md)
+
+### Bug fixes
+
+* expose (additional) Python commands in the synapser package ([SYNR-1474](https://sagebionetworks.jira.com/browse/SYNR-1474))
```

Comment:
My only suggestion would be to capitalize "Expose" as the first word of this bullet point.